components/asp/database/EventWriter.py

```
@@ -0,0 +1,68 @@
+        '''
+        if hasattr(self, 'conn') and self.conn:
+            self.conn.close()
+
+    def checkIfFieldExists(self, table, column, value):
+        '''
+            Checks if the database has the given field.
+        '''
+        query = f'SELECT {column} FROM {table} WHERE {column} = ?' 
+        self.cursor.execute(query, [value])
+        row = self.cursor.fetchone() 
+        return not (row == None)
```

Ruff (0.8.2)

Use `row != None` instead of `not row == None`

Replace with `!=` operator

(SIM201)